index.html

```diff
@@ -0,0 +1,205 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <link rel="icon" type="image/svg+xml" href="favicon.svg" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+  <title>Tech News Page</title>
+  <link rel="stylesheet" href="https://use.typekit.net/kjf8squ.css">
+  <script src="https://kit.fontawesome.com/67c66657c7.js"></script>
+  
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<!-- settings and social media nav starts here -->
+
+<body>
+  <!-- <div class="first-nav-wrap">
+    <div class="container first-nav">
+      <ul class="settings-and-social-list">
+        <li><a href="#"><img src="img/twitterlogo.png" alt="twitter-icon"></a></li>
+        <li><a href="#"><img src="img/linkedinlogo.png" alt="linkedin-icon"></a></li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My page</a></li>
+          <li><a href="#">Logout</a></li>
+      </ul>
+     </div>
+    </div>
+   -->
```

Comment:
- [ ] **Optional:** Please remove all commented lines to make your code more clean.

Reply:
section removed